src/components/content/videos/VideoDetails.js

```
@@ -15,228 +15,377 @@ import useWindowWidth from "@/hooks/useWindowWidth";
+  const handleDelete = async () => {
+    try {
+      const response = await axios.delete(`/api/resources/${processedEvent.d}`);
+      if (response.status === 204) {
+        showToast("success", "Success", "Resource deleted successfully.");
+        router.push("/");
+      }
+    } catch (error) {
+      if (
+        error.response &&
+        error.response.data &&
+        error.response.data.error.includes("Invalid `prisma.resource.delete()`")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)